### PR TITLE
Display articles hierarchically

### PR DESCRIPTION
**Pull request by rasendubi**
Allow editing article parent and display articles in a hierarchy.

2018-01-28-193714_2141x875_escrotum
2018-01-28-193730_1521x1232_escrotum

### CODE REVIEW

File: src/components/ArticlesList/ArticleSelect.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/* eslint-disable */
```
Comment (drets) [inline src/components/ArticlesList/ArticleSelect.js:1]: :open_mouth:

**Comment by drets** [comment]
image

Could we add a label for parent article?

**Comment by drets** [comment]
Do we really need to show a parent id in database on frontend when parent article selected?

File: src/components/ArticlesList/ArticleSelect.js (hunk 1 above)
Comment (rasendubi) [reply src/components/ArticlesList/ArticleSelect.js:1]: oops. will fix now

**Comment by rasendubi** [comment]
> Do we really need to show a parent id in database on frontend when parent article selected?

I want to show something as it helps to understand if any bookmark is selected as a parent. (Currently, if you edit the text after selection, no parent will be chosen.)

**Pull request merged**